nav2_behavior_tree/plugins/decorator/goal_updater_node.cpp

```diff
@@ -46,7 +46,7 @@ GoalUpdater::GoalUpdater(
   sub_option.callback_group = callback_group_;
   goal_sub_ = node_->create_subscription<geometry_msgs::msg::PoseStamped>(
     goal_updater_topic,
-    10,
+    rclcpp::SystemDefaultsQoS(),
```

Comment:
Tested with cyclonedds. SystemDefaultsQoS has KeepLast policy by default. Maybe we should make that explicit in case some other vendor doesn't have that as default?